install.md

```diff
@@ -28,7 +28,7 @@ Once you have installed all the [prerequisites](install-prerequisites.md), make
 
     ```bash
     tanzu package repository add tas-adapter-repository \
-      --url dev.registry.tanzu.vmware.com/app-service-adapter/tas-adapter-package-repo:latest \
+      --url registry.tanzu.vmware.com/app-service-adapter/tas-adapter-package-repo:0.1.0-build.1 \
```

Comment:
@davewalter I was having secret issues in general and switched to this before I had resolved them, but I still think it's required. Let me know if it's not.